corehq/motech/repeaters/migrations/0001_adjust_auth_field_format.py

```diff
@@ -1,31 +1,14 @@
-from __future__ import absolute_import
-from __future__ import unicode_literals
-from django.db import migrations
-
-from corehq.motech.repeaters.models import Repeater
-from corehq.motech.repeaters.utils import migrate_repeater
-
-from corehq.util.couch import iter_update, DocUpdate
+from __future__ import absolute_import, unicode_literals
 
+from django.db import migrations
 
-def migrate_auth_field(apps, schema_editor):
-    repeater_ids = [row['id'] for row in Repeater.view(
-        'repeaters/repeaters',
-        include_docs=False,
-        reduce=False,
-        wrap_doc=False
-    )]
-    iter_update(
-        db=Repeater.get_db(),
-        fn=migrate_repeater,
-        ids=repeater_ids,
-    )
+from corehq.util.django_migrations import noop_migration
 
 
 class Migration(migrations.Migration):
 
     dependencies = []
 
     operations = [
-        migrations.RunPython(migrate_auth_field),
+        noop_migration(),
```

Comment:
I think that these don't actually need to be `noop_migration`. I believe that they can be 

```operations = []```

Reply:
I thought Django required an operation, was going off of this:
https://github.com/dimagi/commcare-hq/blob/f3f04690be1581c9e11806df2a3cd56c166ca6d4/corehq/form_processor/README.md#L98-L100

Are you saying there's never any reason to use `noop_migration()`?

Reply:
offlined with @emord.  Looks like that is indeed the case.  Will update this PR accordingly.